src/BlazorIdentity/UserLoginInfoResult.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace BlazorIdentity;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+public class UserLoginInfoResult
```

Comment:
Let's just keep it the same name as the type it's shimming, like the other shim types in this library.